doc/sphinxdoc/generate_reference.py

```diff
@@ -160,6 +187,15 @@ def doc2rst(algo_doc, sphinxsearch=False):
         lines = [ algo_doc['name'], '=' * len(algo_doc['name']), '']
         lines += [algo_doc['mode'] + ' | ' + algo_doc['category'] + ' category', '']
 
+    links = source_links(algo_doc)
+    if links:
+        lines += ['Source Code',
```

Comment:
"Source code" - for consistency in capitalization in the documentation (e.g., see "See also" in the same algorithm page).